src/Emr/Eloquent/ProviderData.php

```diff
@@ -84,12 +73,17 @@ public function setAddress($address)
 
     public function getName()
     {
-        // TODO @leo: Implement getName() method.
+        return $this->practice_name;
     }
 
-    public function getCode()
+    public function setName($practiceName)
     {
-        // TODO @leo : Implement getCode() method.
+        $this->practice_name = $practiceName;
+        return $this;
+    }
 
+    public function getCode()
+    {
+        return $this->getId();
```

Comment:
get Code should return a string representation of a JSON object that contains data the the client needs, such as this:
`{
    "id": "57e3fa727e77bb26ee9166b0",
    "name": "Bolton Gomez",
    "practiceName": "Lavonne Practice",
    "address": "303 Bay Street, Beyerville, Texas"
  }`